documentation/rfc/journeys_ridesharing.json

```diff
@@ -0,0 +1,556 @@
+                                        "value": "0087-318964-BV"
+                                    }
+                                ],
+                                "name": "Avignon-TGV",
+                                "links": [],
+                                "coord": {
+                                    "lat": "43.921963",
+                                    "lon": "4.78616"
+                                },
+                                "label": "Avignon-TGV (Avignon)",
+                                "timezone": "Europe/Paris",
+                                "id": "stop_area:OCE:SA:87318964"
+                            }
+                        },
+                        "quality": 0,
+                        "name": "Avignon-TGV (Avignon)",
+                        "id": "stop_point:OCE:SP:TGV-87318964"
+                    },
+                    "co2_emission": {
+                        "value": 0,
+                        "unit": ""
+                    },
+                    "geojson": {
+                        "type": "LineString",
+                        "properties": [
+                            {
+                                "length": 138097
+                            }
+                        ],
+                        "coordinates": [
+                            [
+                                3.077759,
+                                44.10067
+                            ],
+                            [
+                                4.78616,
+                                43.921963
+                            ]
+                        ]
+                    },
+                    "duration": 9763,
+                    "type": "crow_fly",
+                    "id": "section_6_0",
+                    "mode": "ridesharing",
+                    "ridesharing_journeys": [
+                        {
+                            "fare": {
+                                "found": false,
+                                "links": [
+                                    {
+                                        "id": "ticket_1",
+                                        "internal": true,
+                                        "rel": "tickets",
+                                        "templated": false,
+                                        "type": "ticket"
+                                    }
+                                ],
+                                "total": {
+                                    "value": "12.40",
+                                    "currency": "EUR"
+                                }
+                            },
+                            "sections": [
+                                {
+                                    "from": {
+                                        "name": "Millau (12100)",
+                                        "id": "admin:fr:12145"
+                                    },
```

Comment:
Objects should be valid ones to facilitate integration

Reply:
Yep, I just didn't develop that.
OK to add more info from what you successed to import in playground